frontend/src/components/maps/leaflet/InfoSlideOut/InfoContent.tsx

```diff
@@ -113,8 +117,21 @@ export const InfoContent: React.FC<IInfoContent> = ({
             leftSideLabel={'Classification'}
             rightSideItem={propertyInfo?.classification}
           />
+          {!!propertyInfo?.projectNumbers?.length && (
+            <ThreeColumnItem
+              leftSideLabel={`Project Number${propertyInfo.projectNumbers.length > 1 ? 's' : ''}`}
```

Comment:
I think I'd be fine with just Project Number(s) in the future. I don't think the logic here is worth the overhead.

Reply:
not wrong ofc.